### PR TITLE
Tape tests

### LINKED ISSUE #9: Add testing
_From @BerkeleyTrue on October 30, 2015 18:51_

_Copied from original issue: FreeCodeCamp/ChallengeOMatic1003#7_

### PR DESCRIPTION
**Pull request by terakilobyte**
Closes #9

### CODE REVIEW

File: tests/reducers.spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      'view': 'ChallengeSelect'
+    }
+  };
+  return fixtures;
+
+};
+
+
+test('should return a default state if none is provided', (t) => {
+  let fixtures = setup();
```
Comment (BerkeleyTrue) [inline tests/reducers.spec.js]: Use TDD not BDD

`Test 1 should return a default state if none is provided`

**Comment by terakilobyte** [comment]
Updated.

**Comment by SaintPeter** [comment]
Can't seem to get this to work on Windows.  Will try again with Cygwin.

**Comment by SaintPeter** [comment]
Tested in CygWin:

```
Rex@Rextastic /cygdrive/d/mean/COM1000
$ npm test

> COM1000@0.0.0 test D:\mean\COM1000
> node node_modules/.bin/babel-tape-runner tests/**/*.js | faucet

D:\mean\COM1000\node_modules\.bin\babel-tape-runner:4
# case `uname` in
^

SyntaxError: Unexpected token ILLEGAL
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:414:25)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:136:18)
    at node.js:963:3
not ok 1 no plan found
not ok 2 no assertions found
⨯ fail  2
npm ERR! Test failed.  See above for more details.
```

No idea what is going on here.

**Comment by terakilobyte** [comment]
Yeah not sure, I'd suggest opening an issue against tape/babel-tape-runner that their software isn't working under windows.

**Pull request merged**